docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -111,7 +111,7 @@ requirements:
   feature, the change should also contain documentation.
 
 When you think your work is ready to be reviewed, send :doc:`a GitHub pull
-request <working-with-git>`. 
+request <working-with-git>`.
```

Comment:
This was automatically updated from my VSCode. I am happy to revert it back if the change is unnecessary.